src/main/java/dwbh/api/fetchers/UserGroupFetcher.java

```diff
@@ -63,7 +65,7 @@ public UserGroupFetcher(UserGroupService service) {
    */
   public DataFetcherResult<Boolean> addUserToGroup(DataFetchingEnvironment env) {
     Context ctx = env.getContext();
-    UserGroupInput input = GroupFetcherUtils.userGroup(env);
+    UserGroupIdsInput input = GroupFetcherUtils.userGroup(env);
```

Comment:
Doesn't make sense to get the user information twice: once in the input, and then in the the fetcher again. I would say to use just the input. Check the service because it's also redundant.

Reply:
I think If everything required for the service call were retrieved inside `GroupFetcherUtils.userGroup(env)` there will be less room for redundancy.